.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,33 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
please change this to allow the workflow to be executed on push to any branches, this workflow does ui testing and code style checks so it should not be bounded to an specific branch.

```yml

on: [push]

```

Reply:
done